datafusion/src/physical_plan/mod.rs

```diff
@@ -640,6 +640,7 @@ pub mod sort;
 pub mod sort_preserving_merge;
 pub mod stream;
 pub mod string_expressions;
+pub(crate) mod tdigest;
```

Comment:
Is there a reason this is `pub(crate)` while everything else is open?

Reply:
I modelled `approx_quantile()` after `approx_count()` which uses `HyperLogLog` internally - the `hyperloglog` mod is also `pub(crate)` so I did the same - I can change tdigest to `pub` if that makes more sense?

Should `hyperloglog` also be `pub`?

Reply:
I think `pub(crate)` is a good place to start, and we can make it `pub` later on if there is some usecase